DESCRIPTION

```diff
@@ -29,8 +29,7 @@ Suggests: knitr,
     rgeos,
     sf,
     sp,
-    tidyr,
-    tmap
+    tidyr
```

Comment:
That would be my preference. We can do ad hoc examples with other approaches (tmap, leaflet, htmlwidgets, etc.) but they don't need to be packaged vignettes.